cmd/src/config.rs

```diff
@@ -0,0 +1,166 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
+
+use std::error::Error;
+use std::fs;
+use std::io::{Error as IoError, Write};
+use std::path::Path;
+
+use tikv::config::TiKvConfig;
+
+const LAST_CONFIG_FILE: &str = "last_tikv.toml";
+
+#[derive(Clone, Serialize, Deserialize, PartialEq, Debug)]
+#[serde(default)]
+#[serde(deny_unknown_fields)]
+#[serde(rename_all = "kebab-case")]
+pub struct Config {
+    #[serde(flatten)]
```

Comment:
Should we move serde functions from `TiKVConfig` to here? If we add more fields in `tikv.toml`, they belong to `TiKVConfig` or this `Config`? Personally I think it's better to deserialize other component configs from `other_component.toml`, and just combine them into `Config`.

Reply:
> Should we move serde functions from TiKVConfig to here?

I don't think so, serde functions should stay the same mod as `TikvConfig`. 

> If we add more fields in tikv.toml, they belong to TiKVConfig or this Config? 

It depends, if it belongs to `TikvConfig` then it should stay in `TikvConfig`.

> Personally I think it's better to deserialize other component configs from other_component.toml, 

Well, I don't think it's a good idea to introduce multiple config files.

> and just combine them into Config.

Do you mean the `TikvConfig`?